roles/kubernetes/master/templates/manifests/kube-controller-manager.manifest.j2

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: kube-controller-manager
+  namespace: kube-system
+spec:
+  hostNetwork: true
+  containers:
+  - name: kube-controller-manager
+    image: {{ hyperkube_image.name }}:{{ hyperkube_image.tag }}
```

Comment:
I suggest to use following var names for images: 

hyperkube_image:  repo:tag
or 
hyperkube_image_repo:  repo
hyperkube_image_tag: tag